src/app/layout.tsx

```diff
@@ -1,25 +1,25 @@
-import type { Metadata } from 'next'
+'use client'
 
 import Providers from '@/store/Providers'
+import { FC } from 'react'
+import { Toaster } from 'react-hot-toast'
 
 import { text, title } from './fonts'
 import './globals.css'
 
-export const metadata: Metadata = {
-  description: 'Generated by create next app',
-  title: 'Create Next App',
+interface PageProps {
+  children: React.ReactNode
```

Comment:
Personally I always import something from react and I've already did it in code so I think it would be much better to import ReactNode from react
```suggestion
  children: ReactNode
```